x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/action/PutDataFrameTransformAction.java

```diff
@@ -58,6 +58,11 @@ public Request(DataFrameTransformConfig config) {
         public Request(StreamInput in) throws IOException {
             super(in);
             this.config = new DataFrameTransformConfig(in);
+            if (in.getVersion().onOrAfter(Version.CURRENT)) {
+                this.deferValidation = in.readBoolean();
+            } else {
+                this.deferValidation = false;
```

Comment:
In case you decide not to make "deferValidation" member variable final, you can remove the "else" branch here.